acceptance/Rakefile

```diff
@@ -85,7 +85,7 @@ def beaker_test(mode = :packages, options = {})
   if mode == :git
     # Build up project git urls based on git server and fork env variables or defaults
     final_options[:install].map! do |install|
-      if md = /^(\w+)#(\w+)$/.match(install)
+      if md = /^(\w+)#([\d.\w]+)$/.match(install)
```

Comment:
Will this continue to work if we move back to `stable`? Can we make a regex that'll match any branch name?